src/ResourceManager/EventGrid/Commands.EventGrid/Commands.EventGrid.Netcore.csproj

```diff
@@ -28,7 +28,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.EventGrid" Version="2.0.0-preview" />
+    <Reference Include="Microsoft.Azure.Management.EventGrid">
```

Comment:
@ahamad-MS revert the change made in this file and then just change `2.0.0-preview` to `3.0.0-preview`